src/ui/public/vis/editors/default/default.js

```diff
@@ -138,8 +137,23 @@ const defaultEditor = function ($rootScope, $compile) {
           updateScope();
         }
 
-        const visualizationEl = this.el.find('.vis-editor-canvas')[0];
-        visualizationLoader(visualizationEl, this.vis, visData, uiState, { listenOnChange: false });
+        if (!this._handler) {
+          const visualizationEl = this.el.find('.vis-editor-canvas')[0];
+          getVisualizeLoader().then(loader => {
+            this._loader = loader;
```

Comment:
As happen in `vis_editor_visualization.js` line 74 it's better to add a check if `visualizationEl` exists before continuing with this function.